web/src/user_groups.ts

```diff
@@ -370,6 +370,7 @@ export function get_direct_subgroups_of_group(target_user_group: UserGroup): Use
     for (const subgroup_id of subgroup_ids) {
         const subgroup = user_group_by_id_dict.get(subgroup_id);
         assert(subgroup !== undefined);
+        subgroup.name = get_display_group_name(subgroup.name);
```

Comment:
Is this mutating the original object in the data store? I think so.

Reply:
Yes, you are right. On a second thought, it also didn't feel right for a function called `get_direct_subgroups_of_group` to do the display name conversion by default, even if the function is only used in one place right now. I added another function instead to just convert to display name for multiple groups when required. LMK if we want to do the conversion in the original function itself or maybe have display_name as an additional property instead,.